homeassistant/components/omnilogic/strings.json

```
@@ -0,0 +1,22 @@
+{
+  "title": "Omnilogic",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "polling_interval": "[%key:common::config_flow::data::polling_interval"
```

I don't think such a key exists (only the most commonly translated strings have these). You can use plain English here.

---

But I should enter it in en.json so that multiple languages can be handled in future correct?

---

just enter it here, e.g.:
```suggestion
          "polling_interval": "Polling interval (in seconds)"
```

---

You should never edit en.json yourself.

---

ok